client/src/components/MenuBar.js

```diff
@@ -0,0 +1,48 @@
+import Container from 'react-bootstrap/Container';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import NavDropdown from 'react-bootstrap/NavDropdown';
+
+const MenuBar = () => {
+    return (
+        <Navbar bg="light" expand="lg">
+        <Container>
+            <Navbar.Brand href="#home">H-A-R</Navbar.Brand>
```

Comment:
Hi Anthony just a note that the title on the navbar should be H-A-R-H

Reply:
Thanks Tersia, I am missing an H.